recipes/motus/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: motus
+  version: "1.0"
+
+source:
+  fn: mOTUs.Linux64bits.tar.gz
+  url: http://www.bork.embl.de/software/mOTU/share/mOTUs.Linux64bits.tar.gz
```

Comment:
Sure, you can have as many commands as possible in the test section.
For you second question, this is certainly possible but it would be $PREFIX/opt/ and a link to $PREFIX/bin.

An other idea would be to run this tool once in the build.sh script and create the binaries in this step. Then we can remove the binary blob from the pl script.